docs/guest.md

```diff
@@ -184,4 +184,10 @@ CloneFromSnapshot CRD has some key phases for the `.status.phase` field:
 * New: clone from snapshot is not completed
 * InProgress: the vSphere volume snapshot is being downloaded from remote repository.
 * Completed: clone from snapshot is completed
-* Failed: clone from snapshot is failed
\ No newline at end of file
+* Failed: clone from snapshot is failed
+
+### StorageClass Mapping
+Velero plugin for vsphere v1.1.0 and above supports changing the storage class persistent volumes and persistent volume claims during restores, 
```

Comment:
the storage class persistent volumes and persistent volume claims -> the storage class of persistent volumes